featurebyte/routes/scd_table/controller.py

```diff
@@ -70,3 +79,11 @@ async def get_info(self, document_id: ObjectId, verbose: bool) -> SCDTableInfo:
             end_timestamp_column=scd_table.end_timestamp_column,
             current_flag_column=scd_table.current_flag_column,
         )
+
+    async def service_and_query_pairs_for_checking_reference(
+        self, document_id: ObjectId
+    ) -> List[Tuple[Any, QueryFilter]]:
+        return [
+            (self.entity_service, {"primary_table_ids": document_id}),
+            (self.feature_service, {"table_ids": document_id}),
+        ]
```

Comment:
This seems common among all types of tables except for event table. Would it make sense to have this in the base controller class and override / extend it in for even table?

Reply:
It makes sense. Will update this accordingly. 👍